File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.java
class Solution {
    public int solution(String binomial) {
        int answer = 0;
        String[] nums = binomial.split(" ");
        if("+".equals(nums[1])){
            answer = Integer.valueOf(nums[0]) +  Integer.valueOf(nums[2]);
        }else if("-".equals(nums[1])){
            answer = Integer.valueOf(nums[0]) -  Integer.valueOf(nums[2]);
        }else{
            answer = Integer.valueOf(nums[0]) *  Integer.valueOf(nums[2]);
        }
        return answer;
    }
}

File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.cpp
#include <string>
#include <vector>
#include <sstream>
using namespace std;

int solution(string binomial) {
    int answer = 0;
    istringstream ss(binomial);
    string tkn = "";
    string nums[3];
    int i=0;
    while( getline(ss,tkn, ' ' )){
        nums[i++] = tkn;
    }
    if(nums[1] == "*"){
        answer = stoi(nums[0]) * stoi(nums[2]);
    }else if(nums[1] == "+"){
        answer = stoi(nums[0]) + stoi(nums[2]);
    }else{
        answer = stoi(nums[0]) - stoi(nums[2]);
    }
    return answer;
}